### PR TITLE
Revert part of "Merge PR #389"

### PR DESCRIPTION
**Pull request by modos189**
As it turned out, commit [findPortalGuidByCacheE6](https://github.com/IITC-CE/ingress-intel-total-conversion/pull/389/commits/046e038c222bbe715839b00cf6fba3d0eadfcb41) contains [an error](https://github.com/IITC-CE/ingress-intel-total-conversion/pull/388#discussion_r545451398)

An [improved replacement](https://github.com/IITC-CE/ingress-intel-total-conversion/pull/409) is suggested for commit [Issues #364 & hook for Alerts](https://github.com/IITC-CE/ingress-intel-total-conversion/pull/389/commits/cbb38684b0122d18ae7a9a2c8d136d4bb4844a62)

Finally, commit [Change requests](https://github.com/IITC-CE/ingress-intel-total-conversion/pull/389/commits/b8211c82d9df9bcf5c9f917ed73a791fc2d70586) is a hodgepodge of fixes to the commits listed above.

The listed commits were reverted.
But commits [SYNC (authorize)](https://github.com/IITC-CE/ingress-intel-total-conversion/pull/389/commits/5bd9eb9f495da586658ed0ab18bada29a3d9db80) and [BOOKMARKS (markers)]() were retained

### CODE REVIEW

File: plugins/bookmarks.js [hunk 1]
```diff
@@ -991,6 +991,8 @@ window.plugin.bookmarks.loadStorageBox = function() {
       // All data is replaced if other client update the data during this client offline,
       if(fullUpdated) {
         window.plugin.bookmarks.refreshBkmrks();
+        window.plugin.bookmarks.resetAllStars();
+        console.log('BOOKMARKS: synchronized all after offline');
```
Comment (johnd0e) [inline plugins/bookmarks.js]: Perhaps that should be removed

File: plugins/bookmarks.js [hunk 2]
```diff
@@ -1003,8 +1005,9 @@ window.plugin.bookmarks.loadStorageBox = function() {
         delete window.plugin.bookmarks.updateQueue[e.property];
         window.plugin.bookmarks.storeLocal(window.plugin.bookmarks.UPDATE_QUEUE);
         window.plugin.bookmarks.refreshBkmrks();
+        window.plugin.bookmarks.resetAllStars();
         window.runHooks('pluginBkmrksSyncEnd', {"target": "all", "action": "sync"});
-        console.log('BOOKMARKS: synchronized all');
+        console.log('BOOKMARKS: synchronized all from remote');
```
Comment (johnd0e) [inline plugins/bookmarks.js]: And here

**Review by johnd0e — CHANGES_REQUESTED** [review]

File: plugins/sync.js [hunk 3]
```diff
@@ -561,53 +561,46 @@ window.plugin.sync.Authorizer.prototype.authComplete = function() {
...
-    if(authResult && !authResult.error) {
-      _this.authorized = true;
-      plugin.sync.logger.log('all', 'Authorized');
-    } else {
-      _this.authorized = false;
-      var error = (authResult && authResult.error) ? authResult.error : 'not authorized';
-      plugin.sync.logger.log('all', 'Authorization error: ' + error);
-      if (error === "idpiframe_initialization_failed") {
-        plugin.sync.logger.log('all', 'You need enable 3rd-party cookies in your browser or allow [*.]google.com');
+  const doAuth = prompt => gapi.auth2.authorize({
```
Comment (johnd0e) [inline plugins/sync.js]: No much sense in using `const` in single place of 25KB file.

**Review by johnd0e — CHANGES_REQUESTED** [review]

File: plugins/bookmarks.js (hunk 1 above)
Comment (MysticJay) [reply plugins/bookmarks.js]: the "console.log" can be removed.

**Comment by MysticJay** [comment]
@modos189 in PR#389 in file cache-portals-on-map.js only one line is wrong.
unless we have a replacement for the other changes this should not be merged

**Pull request merged**